src/msg_conversions.cpp

```diff
@@ -106,16 +106,19 @@ geometry_msgs::Point set_ros_point(const double & x, const double & y, const dou
   return v;
 }
 
-geometry_msgs::Point pcl_to_ros_vector(const pcl::PointXYZ& pt) {
+geometry_msgs::Point pcl_to_ros_point(const pcl::PointXYZ& pt) {
   return set_ros_point(pt.x, pt.y, pt.z);
 }
 
-Eigen::Vector3d tf_vector3_to_eigen_vector(const tf::Vector3 & v) {
-  return Eigen::Vector3d(v.getX(), v.getY(), v.getZ());
+geometry_msgs::Point tf_vector3_to_ros_point(const tf2::Vector3 & v) {
+  return set_ros_point(v.getX(), v.getY(), v.getZ());
 }
 
-geometry_msgs::Point tf_vector3_to_ros_point(const tf::Vector3 & v) {
-  return set_ros_point(v.getX(), v.getY(), v.getZ());
+tf2::Transform ros_pose_to_tf2_transform(const geometry_msgs::Pose &pose) {
+  tf2::Vector3 pos(pose.position.x, pose.position.y, pose.position.z);
+  tf2::Quaternion q(pose.orientation.x, pose.orientation.y, pose.orientation.z, pose.orientation.w);
+  tf2::Transform tf2_transform(q, pos);
```

Comment:
Any reason you don't do this in one line? I think the compiler will optimize it either way, but just curious.

Reply:
I did it most for readability, as if I got it all in one line it would go over the 120 character limit of `roslint`... It would need at least 3 lines to get it all together, so there wasn't much to gain